docs/getting_started/execution-modes.rst

```diff
@@ -159,6 +164,38 @@ Example DAG:
             "secrets": [postgres_password_secret],
         },
     )
+AWS_EKS
+----------
+
+The ``aws_eks`` approach is very similar to the ``kubernetes`` approach, but it is specifically designed to run on AWS EKS clusters.
+It uses the `EKSPodOperator <https://airflow.apache.org/docs/apache-airflow-providers-amazon/2.2.0/operators/eks.html#perform-a-task-on-an-amazon-eks-cluster>`_
```

Comment:
```suggestion
It uses the `EKSPodOperator <https://airflow.apache.org/docs/apache-airflow-providers-amazon/8.19.0/operators/eks.html#perform-a-task-on-an-amazon-eks-cluster>`_
```

Reply:
I think and let the CI run and complete since it takes time and then we can push this change.